components/brave_rewards/core/endpoint/bitflyer/post_oauth/post_oauth_bitflyer.cc

```diff
@@ -110,17 +111,16 @@ void PostOauth::Request(const std::string& external_account_id,
   request->headers = RequestAuthorization();
   request->content_type = "application/json";
   request->method = mojom::UrlMethod::POST;
-  request->skip_log = true;
 
-  engine_->LoadURL(std::move(request),
-                   base::BindOnce(&PostOauth::OnRequest, base::Unretained(this),
-                                  std::move(callback)));
+  engine_->Get<URLLoader>().Load(
+      std::move(request), URLLoader::LogLevel::kBasic,
```

Comment:
Should this be `kNone` instead?

Reply:
Perhaps. The current implementation would effectively skip logging of both requests and responses, but it seems like it's useful to still log the basic info (URL/method). Open to not logging anything here to match current behavior though. Probably the safest approach.

Reply:
I've updated this and similar calls to use `kNone`, to err on the side of caution. Great catch!

Reply:
Yeah, I saw `skip_log` being mapped to both `kNone`, and `kBasic`, so I thought it's better to ask. As long as we don't log sensitive stuff, I'm good with either, but might be safest to stick with what we have, as you said. 🙂